src/main.js

```diff
@@ -1 +1,14 @@
 console.log('Hello, world!');
+const amount = document.querySelector('.form__total span');
+// console.log(amount);
+const paypalLink = document.querySelector('a');
```

Comment:
Dangerous move if there are other `<a>` on the page — for example, menu items 😱

Since the class name is unique, I would do:

```javascript
const paypalLink = document.querySelector('a.form__cc-heading');
```